lib/cog/adapters/hipchat/api.ex

```diff
@@ -14,7 +14,7 @@ defmodule Cog.Adapters.HipChat.API do
   end
 
   def lookup_room("@" <> handle) do
-    {:ok, user} = lookup_user(name: "@" <> handle)
+    {:ok, user} = lookup_user(handle: handle)
```

Comment:
I agree it's more readable in Erlang, but I'm not a fan of it in Elixir, since it feels too dense. I can't easily find where one function ends and another begins.